packages/components/src/vue/components/options.vue

```diff
@@ -102,7 +102,8 @@ export default {
   },
   methods: {
     resetAllOptions: function () {
-      this.app.resetAllOptions()
+      this.settings.resetAllOptions()
+      this.app.applyOptions()
```

Comment:
similarly, should settings.resetAllOptions call app.applyOptions?

Reply:
Similar to the above, I wanted `resetAllOptions()` to deal with setting domain only. Having it call `app.applyOptions()` would break that: a setting controller have to have a reference to the App Controller. So then it would be a circular reference: App Controller has a reference to the Settings Controller and vice verse. I did not want to have that. What do you think?

Reply:
ok, yes you're right.